graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/fetchers/InsertMutationFetcher.java

```diff
@@ -15,6 +15,8 @@
  */
 package io.stargate.graphql.schema.cqlfirst.dml.fetchers;
 
+import static io.stargate.graphql.schema.cqlfirst.dml.fetchers.TtlFromOptionsExtractor.getTTL;
```

Comment:
Is this needed?

Reply:
It's used at line 67. That doesn't appear in the diff because it was previously an instance method in a parent class.